web/posts/2018-07-29-how-i-use-stuff-day-one.markdown

```diff
@@ -0,0 +1,184 @@
+---
+title: "How I Use Stuff: Day One"
+tags:
+---
+
+* App: [Day One](http://dayoneapp.com/)
+* Cost:  Free with a [$34.99/year Pro plan](http://help.dayoneapp.com/day-one-2-0/day-one-pricing-features-guide) (at the time of this writing)
+
+My journaling workflow has evolved a _lot_ over the past few years. My very
+first entry in Day One is from June 21, 2012, but I didn't start keeping a
+regular journal till early November 2015, which was shortly after I [quit
+drinking][quitting]. I turned to keeping a journal as a way to organize my
+thoughts and feelings in a place that I knew was private and non-judgmental.
+Since then, my usage has largely stayed the same, although I've been able to
+be much more consistent in the past year. I've also added a number of smaller
+semi-automated entries to my practice.
+
+[quitting]: /quitting
+
+## The Workflow
+As of this writing, I have a grand total of 2,144 entries stored in Day One.
+Those entries are split up across a few different journals:
+
+### Journal (1,191 Entries)
+This is my "main" journal, and is where all of my free-form writing goes. I
+don't have any hard and fast rules about this journal, and it's littered with
+photos of graffiti I see while traveling, dumb jokes that I don't feel like
+posting publicly, and longer form entries. I try to write at _least_ one of
+those long-form entries in here every day.
+
+The format of what I count as my daily long-form entry has gone through
+multiple revisions over time. When I first started keeping a journal, I wrote
+multiple entries each day, each with a specific topic. I wrote one about my
+workday, one for my personal thoughts/feelings, and one for each activity I
+did. That pace wasn’t maintainable, and it became exhausting. I eventually
+whittled that down to just writing a single large daily overview entry, but
+got a bit burned out on that as well.
+
+More recently, I've settled into just writing _something_ in here every day. I
+try hard to not judge myself on length or quality of the entry. Instead, I
+just focus on getting something written down. I've moved away from entries
+that are glorified lists of things I did during the day, and I focus more on
+writing about what I _feel_. One of the biggest benefits of journaling for me
+is being able to look back at where I was years ago and see how things have
+changed. The list-style entries have limited interest as time goes on, but the
+entries that center around my feelings and my general outlook are evergreen.
+
+### Daily Q&A (651 Entries)
+When I started my journaling practice, one of the things I wanted to replicate
+the most was the [Daily Q&A Journal][physical-q-a-journal] that I bought for
+my wife nearly 4 years ago. Answering the same question on the same day year
```

Comment:
The amazon page does give some context as to what this is, but it might be worth adding a quick sentence describing the prompts like:

> The journal prompts you with one question each day.